MONITORING.md

```diff
@@ -0,0 +1,58 @@
+Out of the box, the `kudo-spark-operator` has enabled metrics reporting. 
+By default, it supports integration with the [Prometheus operator](https://github.com/coreos/prometheus-operator).
+
+Prometheus Operator relies on `ServiceMonitor` kind which describes the set of targets to be monitored. 
+KUDO Spark Operator configures `ServiceMonitor`s for both the Operator and submitted Spark Applications automatically 
+when monitoring is enabled.
+
+#### Exporting Spark Operator and Spark Application metrics to Prometheus
+1) Ensure `prometheus-operator` is installed on your Kubernetes cluster.
```

Comment:
Can you provide a link that shows users what instructions are needed to install the prometheus-operator? And some information on how they can get to the prometheus dashboard once installed? 

Or are you expecting users to be able to access the pre-installed prom/grafana charts that come with konvoy?

Reply:
Yes, I'm expecting users install the operator to Konvoy, where all of these tools (Prometheus, Grafana) are installed.
On the other hand, we agreed to make the kudo-spark-operator able to run on any kubernetes cluster. So, it make sense provide link to an instruction how to install the prometheus-operator like [this](https://github.com/coreos/prometheus-operator#quickstart).  My only concern I don't personally tried install `prometheus-operator` on vanilla kubernetes cluster and maybe can't help a user in case of any troubles with it.

Reply:
addressed